pages/Documents/ConversationalAI/Third-PartyBots/MicrosoftDirectLine/advanced-features.md

```diff
@@ -0,0 +1,325 @@
+---
+pagename: Advanced Features
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+subfoldername: Microsoft Direct Line
+permalink: third-party-bots-microsoft-direct-line-advanced-features.html
+indicator:
+---
+
+### Sending Pause/Delay Messages
+
+It is possible to add a delay before regular messages and actions.
+This delay will be added before a message or action provided in this activity will be sent.
+
+```json-doc
+{
+  // ...
+  "type": "message",
+  "text": "",
+  "channelData": {
+    "delay": {
+      "delay": 1,
+      "typing": true
+    }
+  }
+}
+```
+
+Figure 1.1 Activity resulting in a delay with typing indicator
+
+- `delay` This is the number of seconds the bot will wait. These are expected to be only whole numbers for example for one second delay you will write 1 as a value
+- `typing` This property will enable/disable the typing indicator while delay is happening. It is optional; if not provided then the value will be considered as true.
+
+{: .important}
+A pure delay activity without any text or other structured content can e.g. be used if the bot should not respond with any 
+content but an error escalation should be prevented (the connector would assume an issue with the bot if no response at all can 
+be found).
+A pure delay activity can also be used as part of [multiple activity](third-party-bots-microsoft-direct-line-introduction.html#configuration) responses.
```

Comment:
`A pure delay activity can also be used as part of [multiple activities]`

Reply:
changed the sentence a bit now